docs/source/func.migrating.md

```diff
@@ -0,0 +1,201 @@
+# Migrating from functorch to torch.func
```

Comment:
Created On date for this file has changed https://docs-preview.pytorch.org/pytorch/pytorch/155369/func.migrating.html compared to [main](https://docs.pytorch.org/docs/main/func.migrating.html) - this has to be fixed.